models/pathology_nuclei_segmentation_classification/configs/train.json

```diff
@@ -0,0 +1,504 @@
+{
+    "imports": [
+        "$import glob",
+        "$import os",
+        "$import skimage"
+    ],
+    "bundle_root": "$os.getcwd()",
+    "ckpt_dir_stage0": "$os.path.join(@bundle_root, 'models', 'stage0')",
+    "ckpt_dir_stage1": "$os.path.join(@bundle_root, 'models')",
+    "ckpt_path_stage0": "$os.path.join(@ckpt_dir_stage0, 'model.pt')",
+    "output_dir": "$os.path.join(@bundle_root, 'eval')",
+    "dataset_dir": "/workspace/Data/Pathology/CoNSeP/Prepared/",
+    "train_images": "$list(sorted(glob.glob(@dataset_dir + '/Train/*image.npy')))",
+    "val_images": "$list(sorted(glob.glob(@dataset_dir + '/Test/*image.npy')))",
+    "train_inst_map": "$list(sorted(glob.glob(@dataset_dir + '/Train/*inst_map.npy')))",
+    "val_inst_map": "$list(sorted(glob.glob(@dataset_dir + '/Test/*inst_map.npy')))",
+    "train_type_map": "$list(sorted(glob.glob(@dataset_dir + '/Train/*type_map.npy')))",
+    "val_type_map": "$list(sorted(glob.glob(@dataset_dir + '/Test/*type_map.npy')))",
+    "device": "$torch.device('cuda:0' if torch.cuda.is_available() else 'cpu')",
+    "stage": 0,
+    "epochs": 50,
+    "val_interval": 1,
+    "learning_rate": 0.0001,
+    "amp": true,
+    "ckpt_dir": "$@ckpt_dir_stage0 if @stage == 0 else @ckpt_dir_stage1",
+    "network_def": {
+        "_target_": "HoVerNet",
+        "mode": "original",
```

Comment:
Hi @KumoLiu 
Is there any reason that you are using `original` mode and not the `fast` mode?
Considering that the result of this training will be used in the inference, and inference time of `fast` mode is better, do you think that you can change all of them to `fast` mode? Thanks

Reply:
Hi @drbeh, yes I'm ok to use any of them.

Reply:
Perfect, so it would be great if you can change it to "fast" across the bundle. Thanks

Reply:
especially the weights. thanks

Reply:
Have changed the mode to "fast" and also added fast mode metrics, thanks!